pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -206,7 +204,7 @@ enum DashTool {
 
   /// This takes in the string label for a given [DashTool] and returns the
   /// enum for that string label.
-  static DashTool? fromLabel(String label) {
+  static DashTool fromLabel(String label) {
     for (final tool in DashTool.values) {
       if (tool.label == label) return tool;
     }
```

Comment:
use the simplified syntax from above:
`DashTool.values.where((t) => t.label == label).firstOrNull;`